### ISSUE TITLE
Missing `extensions_enabled` in table `config`

### ISSUE THREAD

**Issue by kcjpop** [flarum/core#76]
When trying to run `php artisan flarum:seed` for the first time,
```
[2015-05-13 18:41:51] local.ERROR: exception 'ErrorException' with message 'Invalid argument supplied for foreach()' in /home/an/code/flarum/flarum/system/core/src/Support/Extensions/ExtensionsServiceProvider.php:22
Stack trace:
#0 /home/an/code/flarum/flarum/system/core/src/Support/Extensions/ExtensionsServiceProvider.php(22): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'Invalid argumen...', '/home/an/code/f...', 22, Array)
#1 /home/an/code/flarum/flarum/system/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(494): Flarum\Support\Extensions\ExtensionsServiceProvider->register()
#2 /home/an/code/flarum/flarum/system/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php(75): Illuminate\Foundation\Application->register(Object(Flarum\Support\Extensions\ExtensionsServiceProvider))
#3 /home/an/code/flarum/flarum/system/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(468): Illuminate\Foundation\ProviderRepository->load(Array)
#4 /home/an/code/flarum/flarum/system/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/RegisterProviders.php(15): Illuminate\Foundation\Application->registerConfiguredProviders()
#5 /home/an/code/flarum/flarum/system/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(181): Illuminate\Foundation\Bootstrap\RegisterProviders->bootstrap(Object(Illuminate\Foundation\Application))
#6 /home/an/code/flarum/flarum/system/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(195): Illuminate\Foundation\Application->bootstrapWith(Array)
#7 /home/an/code/flarum/flarum/system/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(92): Illuminate\Foundation\Console\Kernel->bootstrap()
#8 /home/an/code/flarum/flarum/system/artisan(11): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 {main}  
```

**Comment by tobscure** [comment]
This will definitely be fixed as part of implementing a proper ConfigRepository in #75

**Comment by franzliedke** [comment]
It's a problem with the installer, really. I shall fix that very soon.

**Comment by franzliedke** [comment]
Fixed in 1ab1631849a3ce8d76bd0d607e06809044add067. We simply forgot to add the `extensions_enabled` field to the config table.
